src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/movements_out_on_given_day.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
```

Comment:
Feels like we should be returning movementType for when the filter is left blank.

Reply:
Yeah I did already think about this..